keylime/db/keylime_db.py

```diff
@@ -38,14 +41,16 @@ def make_engine(self, service):
 
         self.service = service
 
+        p_sz: Union[str, int]
+        m_ovfl: Union[str, int]
```

Comment:
Uhm this is only because there is no shadowing in Python. I do not know how far do we want to go with the type annotation, but IMHO a more correct approach it to decide for one type and do the conversion later in the code.

In any case I am OK with the current solution.

Reply:
In this case I can fix the 5 and 10 to become strings...

Reply:
Removed the type declarations and changed to "5" and "10".